src/util/performWithRetries.js

```diff
@@ -0,0 +1,23 @@
+import assign from 'lodash/assign';
+import promiseRetry from 'promise-retry';
+
+export default function performWithRetries(
+  perform,
+  errorsToRetry = [],
```

Comment:
Resolved (sorry, just need to type these in to manage the to-do list of various comments - I don't think there's a discrete way to do this in github code review?)